### PR TITLE
Add new k8 docs

### PR DESCRIPTION
**Pull request by katarinasupe**
### Description

I created new k8 docs based on changes we made (created Helm charts repo)

### Pull request type

Please delete options that are not relevant and check the ones that are.

- [x] Documentation improvements

### Checklist:

- [ ] I checked all content with Grammarly
- [ ] I performed a self-review of my code
- [ ] I made corresponding changes to the rest of the documentation
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors

### CODE REVIEW

File: docs/reference-guide/deployment/kubernetes.md [hunk 1]
```diff
@@ -1,112 +1,87 @@
...
 
-If you don't require data persistency or your dataset is static, there is no need to use the StatefulSet workload. Stateful applications are more complex to set up and maintain as they require more attention when handling storage information and security.
+Memgraph Helm charts repository currently contains a chart for [**standalone Memgraph deployment**](#helm-chart-for-standalone-memgraph) as a Kubernetes `StatefulSet` workload, which saves data to persistent storage. 
 
-The Helm chart is configured to use the latest **MemgraphDB** docker image from [Docker Hub](https://hub.docker.com/r/memgraph/memgraph), so if you require a different Memgraph Docker image, be sure the modify the chart. 
+:::note
+Currently available Helm chart uses the latest **Memgraph** Docker image from the [Docker Hub](https://hub.docker.com/r/memgraph/memgraph). For other Memgraph Docker images (Memgraph MAGE or Memgraph Platform), modify the chart accordingly. We are eager to see new pull requests on our [helm charts repository](https://github.com/memgraph/helm-charts).
```
Comment (vpavicic) [inline docs/reference-guide/deployment/kubernetes.md:13]:
```suggestion
The currently available Helm chart uses the latest **Memgraph** Docker image from the [Docker Hub](https://hub.docker.com/r/memgraph/memgraph). For other Memgraph Docker images (Memgraph MAGE or Memgraph Platform), modify the chart accordingly. We are eager to see new pull requests on our [helm charts repository](https://github.com/memgraph/helm-charts).
```

File: docs/reference-guide/deployment/kubernetes.md [hunk 2]
```diff
@@ -1,112 +1,87 @@
...
 
-The provided Helm chart is configured to deploy Memgraph as a Kubernetes **StatefulSet** workload, which saves data to persistent storage, a useful feature when deploying a database. Due to the StatefulSet nature of Memgraph, it is also necessary to define a **PersistentVolumeClaims** to store [the data directory](/reference-guide/backup.md) (/var/lib/memgraph). This enables the data to be persisted even if the pod is restarted or deleted. 
 
-If you don't require data persistency or your dataset is static, there is no need to use the StatefulSet workload. Stateful applications are more complex to set up and maintain as they require more attention when handling storage information and security.
+Memgraph Helm charts repository currently contains a chart for [**standalone Memgraph deployment**](#helm-chart-for-standalone-memgraph) as a Kubernetes `StatefulSet` workload, which saves data to persistent storage.
```
Comment (antejavor) [inline docs/reference-guide/deployment/kubernetes.md:10]:
```suggestion
Memgraph Helm charts repository currently contains a chart for [**standalone Memgraph deployment**](#helm-chart-for-standalone-memgraph) as a Kubernetes `StatefulSet` workload, which is designed for services that require permanent storage, such as databases. 
```

File: docs/reference-guide/deployment/kubernetes.md [hunk 3]
```diff
@@ -1,112 +1,87 @@
...
-kind: PersistentVolumeClaim
-metadata:
-  name: memgraph-log-storage
-spec:
-  accessModes:
-  - ReadWriteOnce
-  resources:
-    requests:
-      storage: 500Mi
+The [Helm chart for standalone Memgraph](https://github.com/memgraph/helm-charts/tree/main/charts/memgraph) is configured to deploy Memgraph as a Kubernetes `StatefulSet` workload, which saves data to persistent storage. Due to the `StatefulSet` nature of Memgraph, it is also necessary to define a `PersistentVolumeClaims` to store [the data directory](/reference-guide/backup.md) (`/var/lib/memgraph`). This enables the data to be persisted even if the pod is restarted or deleted.
```
Comment (antejavor) [inline docs/reference-guide/deployment/kubernetes.md:20]:
```suggestion
Due to the `StatefulSet` nature of Memgraph, it is also necessary to define a `PersistentVolumeClaims` to store [the data directory](/reference-guide/backup.md) (`/var/lib/memgraph`). This enables the data to be persisted even if the pod is restarted or deleted. 
```
Change due to repeating sentence.

**Review by antejavor — CHANGES_REQUESTED** [review]

File: docs/reference-guide/deployment/kubernetes.md [hunk 4]
```diff
@@ -1,112 +1,87 @@
...
+helm repo update
+```
+
+### Install Memgraph
 
+To install Memgraph Helm Chart, run the following command:
 ```
+helm install <release-name> memgraph/memgraph
+```
+Replace `<release-name>` with a name of your choice for the release.
```
Comment (vpavicic) [inline docs/reference-guide/deployment/kubernetes.md:46]:
```suggestion
Replace `<release-name>` with the name of the release you chose.
```

File: docs/reference-guide/deployment/kubernetes.md [hunk 5]
```diff
@@ -1,112 +1,87 @@
...
+`service.port` | Kubernetes service port | `7687`
+`service.targetPort` | Kubernetes service target port | `7687`
+`memgraphConfig` | Memgraph configuration settings | `["--also-log-to-stderr=true"]`
 
- The chart also creates two **PersistentVolumeClaims** for storing the data and log directories. Since Memgraph Docker image has root privileges on the data located on volumes and log directories, it is necessary to set the `runAsUser` to `0` in the `securityContext` section of the pod to override the `memgraph` user from the Docker image. Currently, Memgraph must have root privileges on the volumes. 
+To change the default chart values, provide your own `values.yaml` file during the installation:
+```
+helm install <resource-name> memgraph/memgraph -f values.yaml
+```
+Another way of doing that is by setting the values of appropriate parameters:
```
Comment (vpavicic) [inline docs/reference-guide/deployment/kubernetes.md:70]:
```suggestion
Default chart values can also be changed by setting the values of appropriate parameters:
```

**Pull request merged**